nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkActionValidationHelper.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Funsho David
+ */
+
+package org.nuxeo.ecm.core.bulk;
+
+import org.nuxeo.ecm.core.bulk.message.BulkCommand;
+
+/**
+ * Helper class to create messages when actions are invalid.
+ * 
+ * @since 10.10
+ */
+public class BulkActionValidationHelper {
```

Comment:
These helper methods could be define in interface as default methods, wdyt ?

Reply:
Yes or in the abstract class I suggest we create.